flocker/provision/_digitalocean.py

```diff
@@ -206,6 +209,52 @@ def latest_droplet_kernel(droplet,
     return latest_kernel
 
 
+def change_kernel(node_id, token, kernel_version):
+    """
+    Change the configured kernel on a digitialocean node.
+
+    :param bytes node_id: The id of the digitialocean.
+    :param bytes token: A DigitalOcean v2 API token.
+    """
+    # Import here, so that this can be added to ``flocker.provision`` without
+    # having to install ``pyocean``.
+    import pyocean
+    v2client = pyocean.DigitalOcean(access_token=token)
+    v2droplet = v2client.droplet.get(node_id)
+    do_kernel = get_droplet_kernel(v2droplet, kernel_version)
+
+    retry_on_error(
+        [pending_event],
+        v2droplet.change_kernel, do_kernel.id)
+
+
+def hard_reboot(node_id, token):
+    """
+    Reboot a digitialocean node by powering it off and then back on.
+    This is necessary for a new kernel to be selected.
+
+    :param bytes node_id: The id of the digitialocean.
```

Comment:
digitialocean > digitalocean (also above)

"of the digitalocean", probably needs "node" at the end.